src/esp/assets/ResourceManager.cpp

```diff
@@ -227,15 +461,27 @@ bool ResourceManager::loadGeneralMeshData(const AssetInfo& info,
     MeshMetaData metaData;
     loadTextures(*importer, &metaData);
     loadMaterials(*importer, &metaData);
-    loadMeshes(*importer, &metaData);
+    loadMeshes(*importer, &metaData, shiftOrigin);
```

Comment:
Is the origin shift needed?  That will probably break all of our current episode datasets.  We will also have to make sure we propagate the new origin to the annotations for MP3D.

Reply:
Default shift is false. If shifted, the transform is now stored for query. This is only expected to be used for object meshes, not static scenes.